MockTestUtility/FileUtilityMock.cs

```diff
@@ -46,7 +46,6 @@ public async Task<string> ReadFromFile(string filePathSource)
         public async Task WriteToFile(string fileContents, string filePathSource)
```

Comment:
```suggestion
        public Task WriteToFile(string fileContents, string filePathSource)
```

The async without await is going to add a warning during build